extension/demo/src/modules/api.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+export function fetchPayload (accessToken) {
```

Comment:
Maybe a less generic name so it's clear that this is specifically for sub status.

Also you need to import `API_BASE`.